platform/microprofile/src/test/java/org/trellisldp/microprofile/SmokeIT.java

```diff
@@ -35,7 +35,7 @@
     private static final HttpClient testClient = HttpClients.createDefault();
 
     @Test
-    void findTrellis() throws IOException {
+    public void findTrellis() throws IOException {
```

Comment:
Hm, why `public` here?

Reply:
For junit tests, I typically make the test methods `public`

Reply:
Yeah, why? JUnit doesn't need it (and on a minor note, some IDEs can warn on missing or malformed Javadocs, and making test methods `public` tends to cause a lot of false positives for that).

Reply:
Interesting. I'll revert that, too, though I'll be completely re-implementing the test code before this gets merged anyway.